org.eclipse.buildship.core/src/main/java/org/eclipse/buildship/core/internal/extension/DefaultExtensionManager.java

```diff
@@ -13,13 +13,12 @@
 import java.util.Collection;
 import java.util.List;
 
-import com.google.common.annotations.VisibleForTesting;
-
+import org.eclipse.buildship.core.internal.CorePlugin;
+import org.eclipse.buildship.core.invocation.InvocationCustomizer;
 import org.eclipse.core.runtime.IConfigurationElement;
 import org.eclipse.core.runtime.Platform;
 
-import org.eclipse.buildship.core.internal.CorePlugin;
-import org.eclipse.buildship.core.invocation.InvocationCustomizer;
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
Be careful to not sort their import order.  Looks like you need to update your import order preferences.